configuration/locations/locations.csv

```diff
@@ -135,3 +135,4 @@ Swen Entansif | La Colline,199ed01b-d64f-4d19-aedf-ee37c7a83285,,ICU at La Colli
 Swen Entansif Neonatal | La Colline,85e4216d-7f78-42a1-bf5b-18967147db3e,,NICU at La Colline,La Colline,,
 Travay e Akouchman | La Colline,70048c2e-6d86-4f7c-89ff-d9a317758b27,,Labor and Delivery at La Colline,La Colline,,
 UMI | La Colline,d50694a1-d4b3-4da2-bf84-1c82b9a0f758,,Inite maladi enfeksyon (UMI) at La Colline,La Colline,,
+Sal Timoun | Cange,397935c0-92e0-4d4c-be85-54f4447a8794,,Pediatrics at Cange,Cange,,
```

Comment:
For clarity, instead of adding at the end of the file, can we add it up with the other Cange locations?  It shouldn't affect loading.

Reply:
And not to state the obvious but I assume you generate a new uuid using a uuid generator? (In the past I've sometimes copied an old line and forgotten to update the uuid)

Reply:
I have updated the order as requested, and yes, this is a new uuid.